plugins/PerObjectSettingsTool/ToolBarButton.qml

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) 2021 Ultimaker B.V.
```

Comment:
Err? How is this different from the ToolbarButton already in Cura? Shouldn't it have a different name if it's different? The name is perhaps also a bit strange, since it's not quite a button in the toolbar (but a button in the panel of a tool)

Reply:
This is different. If we would use the `Cura.ToolbarButton` component the buttons would have rounded corners and no border.
![Screenshot from 2022-01-25 23-28-32](https://user-images.githubusercontent.com/6638028/151070376-891140bb-9dcf-440a-81f3-d7fb8dff86eb.png)
While this button has a square shape and does have a border.
![Screenshot from 2022-01-25 23-30-04](https://user-images.githubusercontent.com/6638028/151070548-5076396c-94f7-490d-8942-244bd12d315d.png)
Which looks significantly different in my opinion. I wanted to use a `Cura.SecondaryButton`, however I could not use this as the icon displayed within the button was too tiny and the `Cura.SecondaryButton` does not implement the `checked` property which is required for these buttons.

Reply:
After discussion with @lorenzoromagnoli we decided that the `Cura.ToolbarButton` would fit better from a UI perspective. Going to re-use the `Cura.ToolbarButton` component.

Reply:
https://github.com/Ultimaker/Cura/pull/11336/commits/b51c787f622db64b964767754e932fb20d0ac183